v-next/hardhat-ignition/src/index.ts

```diff
@@ -112,15 +120,22 @@ const hardhatIgnitionPlugin: HardhatPlugin = {
       .setAction(import.meta.resolve("./internal/tasks/visualize.js"))
       .build(),
 
-    task(["ignition", "verify"], "Not implemented yet - to be available soon")
+    task(
+      ["ignition", "verify"],
+      "Verify contracts from a deployment against the configured block explorers",
+    )
       .addPositionalArgument({
         name: "deploymentId",
         type: ArgumentType.STRING,
         description: "The id of the deployment to verify",
       })
       .addFlag({
-        name: "includeUnrelatedContracts",
-        description: "Include all compiled contracts in the verification",
+        name: "blockscout",
+        description: "Verify contracts on Blockscout",
+      })
```

Comment:
Can we remove the blockscout flag for the moment.
We will support only etherscan for the moment directly. That gives us some time to think on how we want the cli to work.